kernel/main.rs

```diff
@@ -5,8 +5,9 @@
            unwind_attributes, zero_one, collections_range, question_mark)]
 #![no_std]
 
-#![allow(deprecated)]
-// #![deny(warnings)]
+// Its me, ur friend. If you ever comment this line, you will end up in hell. Seriously, don't. You
```

Comment:
You can still have code with warnings. You can locally allow warnings by `#[allow(warnings)]`.